pygmt/base_plotting.py

```diff
@@ -1643,4 +1643,4 @@ def text(
                 lib.call_module("text", arg_str)
 
     # GMT Supplementary modules
-    from pygmt.src import meca  # pylint: disable=import-outside-toplevel
+    from pygmt.src import inset, meca  # pylint: disable=import-outside-toplevel
```

Comment:
`inset` is not a supplementary module.

Reply:
Probably best to just delete the `# GMT Supplementary modules` line or change it to `# GMT modules`. I don't think we have to care what category the module is anymore as long as it's used for plotting.

Reply:
I agree, we may even have to mix plotting and non-plotting modules, as `isort` may not allow us to split them into group.s

Reply:
@willschlitzer Please remove `# GMT Supplementary modules`.